instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/railtie.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module ActiveModelSerializers # :nodoc:
+      def self.subscribe
+        Instrumentation.instance.subscribe
+      end
+
+      if defined?(::Rails::Railtie)
```

Comment:
I see what you are doing. Instead of requiring this file only if `Rails::Railtie` is defined you are adding it to this file.

Is this how we do it in other instrumentations?

Reply:
> Is this how we do it in other instrumentations?

No, the other instrumentation that have railties don't appear to have any guard around the Railtie class getting defined. I suspect that we ought to be doing something to guard that.

Reply:
Ok we should file an issue for this and fix that. This kind of inconsistency will come back to bite us.